app/views/articles/_full_comment_area.html.erb

```diff
@@ -2,8 +2,13 @@
   <div id="comments" data-updated-at="<%= Time.current %>">
     <% if @article.show_comments %>
       <div class="comments-container-container">
+        <%# TODO: Fix margins, vertical centering %>
+        <div class="flex justify-between ml-7 mr-8">
+          <span class="fs-xl">Discussion</span>
+          <div id="comment-subscription"></div>
+        </div>
         <div
-          class="comments-container"
+          class="comments-container flex flex-col justify-between ml-7 mr-7"
```

Comment:
```suggestion
          class="comments-container mx-2 s:mx-7"
```

This layout is kinda broken because of CSS we have here: https://github.com/thepracticaldev/dev.to/blob/master/app/assets/stylesheets/comments.scss#L126-L127

^ if you remove these two lines, and update utility classes, we should be good.

So: `.comments-container` as well as the header you have above will always be 100% wide with horizontal margin being equal to `$su-2` (or `$su-7` if it's above `$breakpoint-s`).

Does that make sense?

Reply:
Makes sense. Go with what's there for now and just add appropriate margins, since this feature isn't about reworking the layout of the comments container. That's a battle for another day. 😉